script.js

```diff
@@ -297,6 +320,10 @@ Promise.all([
   fetch(cartoCountiesURI).then(res => {
     if (!res.ok) throw Error("Unable to fetch counties geojson");
     return res.json();
+  }),
+  fetch(cartoNationsURI).then(res => {
+    if (!res.ok) throw Error("Unable to fetch counties geojson");
```

Comment:
change error message to "Unable to fetch **nations** geojson"

Reply:
Done